catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/azure-storage-datalake-component.adoc

```diff
@@ -0,0 +1,119 @@
+[[azure-storage-datalake-component]]
```

Comment:
The documentation is laking a lot of info, please add info about the component, examples, usage, info about the headers that users can use in the producer and expect in the consumer,  any features that you implemented and you want users to try out ... etc. Ex:  https://camel.apache.org/components/latest/azure-storage-blob-component.html